experimental/devices/sn3218/doc.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+//
+// Package sn3218 controls a SN3218 LED driver with 18 LEDs over an i2c bus.
```

Comment:
In the end of the day, the sn3218 is really just 18x PWMs with an unspecified(?) frequency. So while the datasheet calls out LEDs use case, people can use it for anything. So I'd prefer not abuse the word "LED" in the function names below.

Reply:
removed LED from the functions and renamed function parameters from led to channel